libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/view/MapboxSoundButton.kt

```diff
@@ -215,7 +215,7 @@ class MapboxSoundButton : ConstraintLayout {
 
     private companion object {
         const val SLIDE_DURATION = 300L
-        const val EXTEND_MUTED_TO_WIDTH = 140
+        const val EXTEND_MUTED_TO_WIDTH = 150
```

Comment:
Should this be part of https://github.com/mapbox/mapbox-navigation-android/pull/4219?